_posts/2024-08-22-Boost-VectorSearch-With-CSS .md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title:  "Boosting Vector Search Performance With Concurrent Segment Search"
+authors:
+- vijayanb
+- navneev
+- vamshin
+date: 2024-08-22
+categories:
+  - technical-posts
+  - search
+meta_keywords: concurrent segment search, search concurrency, vector search, k-NN
+meta_description: With concurrent segment search enabled, vector search brings 60% improvements in service time by just utilizing additional 25-35% of CPU.
```

Comment:
Please update the meta with the following: 

_meta_keywords:_  concurrent segment search, OpenSearch vector search, shard optimization, k-NN

_meta_description:_  The concurrent segment search feature in OpenSearch optimizes CPU usage and enhances vector search performance by executing queries in parallel across multiple segments within a shard.